### DISCUSSION TITLE
List view - hide color tags and colorize the text.

### DISCUSSION THREAD

**Discussion by rRobis** [SubtitleEdit/subtitleedit#9070] [Ideas]
Would be very nice, if there would be an option, that instead of color tags ({\c&H10E5FF&}sometext{\c&HFFFFFF&}, the text itself would be colorized in the list view and would not show the tags at all.

It would make the list view more readable.
@niksedk

**Comment by Josua-92** [comment]
Oh yes, I'm new to SE, and I was wondering how to do that. The text field alone isn't very helpful if you want to quickly jump to specific timestamps. The List view should definitely show the text with the applied tags, especially when using the 'highlight current word' parameter with audio-to-text (Whisper). It would help to visually locate lines much faster. The way it is now is exhausting and confusing.
